src/helpers.ts

```diff
@@ -10,13 +14,13 @@ export function getCurrentPositionTicks($scope) {
     return positionTicks;
 }
 
-export function getReportingParams($scope) {
+export function getReportingParams($scope: GlobalScope): any {
     /* Math.round() calls:
      * on 10.7, any floating point will give an API error,
      * so it's actually really important to make sure that
      * those fields are always rounded.
      */
-    var volumeInfo = window.castReceiverContext.getSystemVolume();
+    var volumeInfo = window.volume;
```

Comment:
In the code below here we are reporting back the the server the volume of the cast device, so if I understand correctly, we don't need this?

Reply:
...we should probably let jellyfin in on what the chromecast volume is